exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/DatabaseDialect.kt

```diff
@@ -124,20 +132,26 @@ interface DatabaseDialect {
     /** Returns the SQL command that adds a primary key specified [pkName] to an existing [table]. */
     fun addPrimaryKey(table: Table, pkName: String?, vararg pkColumns: Column<*>): String
 
+    /** Returns the SQL command that creates a database with the specified [name]. */
```

Comment:
I noticed the usage of "SQL command" here but "SQL statement" in this [PR](https://github.com/JetBrains/Exposed/pull/1893/files). Just curious if there's any reason for this difference.

Reply:
Sure, that was the old pattern for this file that I followed since there's debate about whether they're interchangeable or whether 'command' actually encompasses both 'statement' and 'query' (and whether even those two should be interchangeable). I'll swap them here and use only statement or query depending on use/result.